hudi-client/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -725,6 +698,13 @@ private synchronized void commit(JavaSparkContext jsc, JavaRDD<HoodieRecord> rec
           throw new HoodieMetadataException("Failed to commit metadata table records at instant " + instantTime);
         }
       });
+      // trigger cleaning, compaction, with suffixes based on the same instant time. This ensures that any future
+      // delta commits synced over will not have an instant time lesser than the last completed instant on the
+      // metadata table.
+      writeClient.clean(instantTime + "001");
```

Comment:
This is a good idea. I hope it does not break some function which may depend on yyyyMMddHHmmss format of the instantTime.

Reply:
@prashantwason our tests use 001,002 etc. So code should be fine. but may be some metrics reporting code might have some issue. let me double check. good call

Reply:
![image](https://user-images.githubusercontent.com/1179324/102947424-2a37db00-4478-11eb-9d90-3d86705d11ca.png)

Seems like the parser can handle this